extensions/healthie/webhooks/appliedTagCreated.ts

```diff
@@ -14,24 +17,33 @@ const dataPoints = {
 
 export const appliedTagCreated: Webhook<
   keyof typeof dataPoints,
-  HealthieWebhookPayload
+  HealthieWebhookPayload,
+  typeof settings
 > = {
   key: 'appliedTagCreated',
   dataPoints,
   onWebhookReceived: async ({ payload, settings }, onSuccess, onError) => {
-    const { resource_id: createdAppliedTagId } = payload
+    try {
+      const { sdk} = await createSdk({settings})
+      const createdAppliedTagId = payload.resource_id.toString()
 
-    if (isNil(createdAppliedTagId)) {
-      await onError({
-        // We should automatically send a 400 here, so no need to provide info
-      })
-    } else {
+      const response = await sdk.getAppliedTag({ id: createdAppliedTagId })
+      const healthiePatientId = response?.data?.appliedTag?.user_id
       await onSuccess({
         data_points: {
           createdAppliedTagId,
         },
+        ...(!isNil(healthiePatientId) && {
+          patient_identifier: {
+            system: HEALTHIE_IDENTIFIER,
+            value: healthiePatientId,
+          },
+        }),
       })
-    }
+    } catch (error) {
+      const formattedError = formatErrors(error)
+      await onError(formattedError)
```

Comment:
this implementation is great--no need to change. personal preference: i'm perfectly fine with `onError(formatError(err))`. although in some cases nested functions can be difficult to read, in this particular case it would read to me very naturally

Reply:
agreed I will update accordingly